From e0d745096444c0ee9b74068d07cb68d79caf07ec Mon Sep 17 00:00:00 2001
From: Tomas Chvatal <tomas.chvatal@gmail.com>
Date: Wed, 24 Jul 2013 22:59:44 +0200
Subject: [PATCH] Fix missing include of iostream
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

As without it the test just happily fails to compile due to this:

tests/zypp/Flags_test.cc:9:12: error: ‘std::cout’ has not been declared
---
 tests/zypp/Flags_test.cc | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/tests/zypp/Flags_test.cc b/tests/zypp/Flags_test.cc
index d6e4666..95dee3c 100644
--- a/tests/zypp/Flags_test.cc
+++ b/tests/zypp/Flags_test.cc
@@ -4,6 +4,7 @@
 #include <boost/test/auto_unit_test.hpp>
 #include "zypp/base/Logger.h"
 #include "zypp/base/Flags.h"
+#include <iostream>
 
 using boost::unit_test::test_case;
 using std::cout;
@@ -80,4 +81,4 @@ BOOST_AUTO_TEST_CASE(flags)
   t.unsetFlag( T_2 ); 		BOOST_CHECK( t.testFlag( T_1 ) );
   t.setFlag( T_1, false );	BOOST_CHECK( t.testFlag( T_0 ) );
   t.setFlag( T_3, true );	BOOST_CHECK( t.testFlag( T_3 ) );
-}
\ No newline at end of file
+}
-- 
1.8.1.5

